2.0/Box/glTF-pbrSpecularGlossiness/Box.gltf

```diff
@@ -112,8 +112,26 @@
                     0.0,
                     0.0,
                     1.0
-                ]
-            }
+                ],
+                "metallicFactor": 0.0
+            },
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [
+                        0.800000011920929,
+                        0.0,
+                        0.0,
+                        1.0
+                    ],
+                    "specularFactor": [
+                        0.20000000298023225,
```

Comment:
> a validation warning

Perhaps. It will be tough for the validator to verify this until you also include math for converting SG to MR (which may or may not be the same for all implementations). Also, would you verify only factors and not textures? I think it will be difficult for the validator to validate visual issues.